scripts/e2e-tests.sh

```diff
@@ -211,6 +211,7 @@ function create_test_cluster() {
     ${extra_flags[@]} \
     ${EXTRA_KUBETEST_FLAGS[@]}
   echo "Test subprocess exited with code $?"
+  (( ! SKIP_ISTIO_ADDON )) && (wait_until_batch_job_complete istio-system || return 1)
```

Comment:
Wrong place. This should be in setup_test_cluster().

Reply:
The PR is WIP and I'm trying to find the root cause, but seems it's not.
From the log it's complaining the `Gateway` CRD of istio and `Image` CRD of knative caching are not found.

Reply:
https://github.com/knative/test-infra/issues/829#issuecomment-496323288